src/main/java/com/thebois/models/beings/Colony.java

```diff
@@ -25,7 +25,7 @@
  */
 public class Colony extends AbstractBeingGroup implements IRoleAllocator, IInventory {
 
-    private final IInventory inventory = new Inventory();
+    private final IInventory colonyInventory = new Inventory();
```

Comment:
style

It's an inventory in a colony. No need to name it `colonyInventory`, just `inventory` is fine.

Reply:
it was hidden by the field in calculateDifference(final IInventory inventory)

Reply:
Good idea, i've changed it